integration-tests/pom.xml

```diff
@@ -62,6 +62,7 @@
         <module>oidc-tenancy</module>
         <module>keycloak-authorization</module>
         <module>reactive-pg-client</module>
+        <module>reactive-pg-client-creds</module>
```

Comment:
Let's use `credentials` instead of `creds` please.

Reply:
the test has been moved to the deployment module as suggested later on